R/track-util.R

```diff
@@ -0,0 +1,142 @@
+#' track helpers
```

Comment:
functions are moved into track.R. You recommend not to use a track-utils.R at all?